src/benchcab/model.py

```diff
@@ -87,7 +87,7 @@ def get_exe_path(self, mpi=False) -> Path:
         exe = internal.CABLE_MPI_EXE if mpi else internal.CABLE_EXE
         if self.install_dir:
             return internal.SRC_DIR / self.name / self.install_dir / exe
-        return internal.SRC_DIR / self.name / self.src_dir / "offline" / exe
+        return internal.SRC_DIR / self.name / "bin" / exe
 
```

Comment:
Won't this break for custom build script? Should we add something in the doc for v2 support that the executable needs to be copied to the bin/ directory?

Reply:
The [`install_dir`](https://benchcab.readthedocs.io/en/latest/user_guide/config_options/#install_dir) key was added to address this.

Reply:
I will mention the `install_key` in the section on "Running with CABLE version 2.x", updating now.